src/tests/ftest/container/multiple_delete.py

```diff
@@ -61,7 +61,7 @@ def test_multiple_container_delete(self):
         self.log.info("SCM = %d, SSD = %d", final_scm_fs, final_ssd_fs)
 
         self.log.info("Verifying SSD space is recovered")
-        self.check_pool_free_space(self.pool, expected_nvme=initial_ssd_fs)
+        self.assertTrue(self.pool.check_free_space(expected_nvme=initial_ssd_fs))
```

Comment:
When triaging functional tests, the `False is not True` error that we get from `assertTrue` is not very useful.  Its preferred to have a meaningful message sent through `self.fail()` that can quickly identify why the test has failed.

Also should you agree to `self.pool.check_free_space()` raising an exception, the code should look like this:
```suggestion
        try:
            self.pool.check_free_space(expected_nvme=initial_ssd_fs)
        except DaosTestError as error:
            self.fail("SSD space is not recovered after 50 create-write-destroy iterations") form error
```